linker/shared/src/main/scala/org/scalajs/linker/backend/emitter/Emitter.scala

```diff
@@ -575,9 +581,7 @@ final class Emitter(config: Emitter.Config) {
               linkedClass.superClass, // invalidated by class version
               linkedClass.jsSuperClass, // invalidated by class version
               useESClass, // invalidated by class version (depends on kind, config and ancestry only)
-              ctor, // invalidated directly
-              memberMethods, // invalidated directly
-              exportedMembers.flatten // invalidated directly
+              ctor ++ memberMethods ++ exportedMembers.flatten // all 3 invalidated directly
```

Comment:
If these are all `List`s, `:::` is more efficient:
```suggestion
              ctor ::: memberMethods ::: exportedMembers.flatten // all 3 invalidated directly
```

Reply:
Changed .